oshdb-util/src/test/java/org/heigit/ohsome/oshdb/util/geometry/relations/OSHDBGeometryBuilderMultipolygonInvalidInnersTest.java

```diff
@@ -25,10 +25,18 @@ public OSHDBGeometryBuilderMultipolygonInvalidInnersTest() {
   }
 
   @Test
-  public void test() {
+  public void testDuplicateInnerRings() {
     // data has invalid (duplicate) inner rings
     OSMEntity entity = testData.relations().get(1L).get(0);
     Geometry result = OSHDBGeometryBuilder.getGeometry(entity, timestamp, tagInterpreter);
     assertTrue(result instanceof Polygon);
```

Comment:
Is it sufficient to test for some polygon? Don't we have to test if it is the polygon we would expect as result? e.g. the polygon in `invalid-inner-rings.osm` with or without the inner hole. Or is this already covered by another test?

Reply:
There should already be "enough" tests covering the basic building of multipolygons with various configurations of inner rings (e.g. see `test750` ff.). So here we only test for the algorithm not choking up on this particular invalid input data.